README.md

```diff
@@ -56,74 +56,99 @@ Documentation on how to package Theia as a Desktop Product may be found [here](h
 
 ### Build
 
+#### Downloading dependencies
+
+To prepare the workspace the dependencies, plugins and the trace-compass server need to be downloaded.
+
+```sh
+yarn
+```
+
+```sh
+yarn download:plugins
+```
+
+```sh
+yarn tracecompass-server:download
+```
+
+#### Build all applications
```

Comment:
As this is the first build instruction users encounter this will likely be the one they execute, again leading them to build 3 apps at once. We should first give an overview of what is contained in the repo in the `### Build` section and then start with the most important one, the electron variant. Then we should mention what the other variants are used for and how to build them. Only then how to build all 3 at the same time.
Also the dev container section should be linked to in `### Build`